app/controllers/summaries_controller.rb

```diff
@@ -10,5 +10,9 @@ class SummariesController < ApplicationController
   # GET /summaries.json
   def index
     @patron = patron
+
+    redirect_to unavailable_path if patron.record.nil?
```

Comment:
Is it only the summaries page where we would want to do this redirect?  Could a user theoretically be on one of the other tabs (e.g. Checkouts) when this occurs?

Reply:
Is there a better name for what is meant w/ `patron.record.nil?`?  Is this a state of our patron that we can give a name and call that?  Like, `patron.fubar?` (not really though 😅)

Reply:
I think the Summaries page is (mostly) right. This should happen as soon as they log in, and it's with the `@patron.record` object that we first know something is amiss. You can't get to one of the other tabs directly if you are not already logged in. But yeah... If you are already logged in and then web services goes down and you then refresh on one of the other tabs, the change here would not capture the disconnect and you would see the generic rails error.

Reply:
I made a helper method called `patron_info_unavailable?`. It could also be called from the other tabs if you thing those 👆 scenarios might be worth capturing.

Reply:
I made a helper method called patron_info_unavailable?. It could also be called from the other tabs if you thing those 👆 scenarios might be worth c

Reply:
I'll leave it up to you if you think it's worth doing to the other controllers.  (and if we did that, it might indicate that the unavailable endpoint might make sense as its own controller)

Reply:
And just to be clear wrt a "helper", my comment is more leaning towards some method being on the patron class that you can ask if the patron is problematic.